ci/supportfiles/sparkpermissions.conf

```diff
@@ -0,0 +1,56 @@
+{
+  "roles": {
+    "default" : {
+      "url" : {
+        "allow" : [ "*" ],
+        "block" : [
+          "*://localhost",
+          "*://localhost:*",
+          "*://127.0.0.1",
+          "*://127.0.0.1:*",
+          "*://[::1]",
+          "*://[::1]:*",
+          "*://[0:0:0:0:0:0:0:1]",
+          "*://[0:0:0:0:0:0:0:1]:*",
+          "file://*",
+          "*://www.pxscene.org"
+        ]
+      },
+      "serviceManager" : {
+        "block" : [ "*" ]
+      },
+      "features" : {
+        "allow" : [ "*" ]
+      },
+      "applications" : {
+        "allow" : [ "*" ]
+      }
+    },
+    "local" : {
+      "url" : {
+        "allow" : [ "*" ]
+      }
+    },
+    "pxscene.org" : {
+      "url" : {
+        "allow" : [ "*" ]
+      }
+    }
+  },
+  "assign" : {
+    "*" : "default",
+    "*://localhost" : "local",
+    "*://localhost:*" : "local",
+    "*://127.0.0.1" : "local",
+    "*://127.0.0.1:*" : "local",
+    "*://[::1]" : "local",
+    "*://[::1]:*" : "local",
+    "*://[0:0:0:0:0:0:0:1]" : "local",
+    "*://[0:0:0:0:0:0:0:1]:*" : "local",
+    "file://*" : "local",
+    "*://www.pxscene.org" : "pxscene.org",
+    "*://www.pxscene.org:*" : "pxscene.org",
+    "*://pxscene.org" : "pxscene.org",
+    "*://pxscene.org:*" : "pxscene.org"
+  }
+}
```

Comment:
Let's avoid committing a default permissions file.  We can use the default permissions that are in code today.